config.yaml

```diff
@@ -72,6 +72,13 @@ clients:
       blockwiseTransfer:
         enabled: true
         blockSize: "1024"
+      ownershipTransfer:
+        method: justWorks
```

Comment:
shouldn't this be now `method: [justWorks]` ?

Reply:
and allowedMethods ?

Reply:
do we want `allowed` ? it seems clear without it.

Reply:
`method: [justWorks]`  ->`methods: [justWorks]`